rockcraft.yaml

```diff
@@ -23,9 +24,6 @@ parts:
     python-packages:
       - wheel
       - pip
-      # Limited to < 66 because we need `pkg_resources` and because `python-apt`
```

Comment:
no idea why this isn't needed anymore :shrug: 
and in fact breaks with the current Python 3.12 distribution in noble

Reply:
as discussed, we do not need this anymore